test/jinja2-when-success.yml

```diff
@@ -3,8 +3,6 @@
     - name: test when
       debug: msg=text
       when: true
-
-- hosts: all
```

Comment:
switching from all to localhost, I assume this is the pattern now

Reply:
It does not matter as long we are using values that are valid with implicit inventory. We do not really run them.